content/20.pharmaceuticals.md

```diff
@@ -651,6 +653,25 @@ Interest in TCZ as a treatment for COVID-19 was supported by two meta-analyses t
 Additionally, different studies used different dosages, number of doses, and methods of administration; ongoing research may be needed to optimize administration of TCZ [@doi:10.1016/j.jcv.2020.104443], although similar results were reported by one study for intravenous and subcutaneous administration [@doi:10/d2pk].
 Clinical trials that are in progress are likely to provide additional insight into the effectiveness of this drug for the treatment of COVID-19 along with how it should be administered.
 
+##### Anakinra
+Anakinra is a recombinant of the IL-1 receptor antagonist (IL-1Ra) and blocks the binding of IL-1α and IL-1β to IL-1R1.
+It is primarily used to reduce inflammation by mediating IL-1, providing a broad spectrum of applications that include the treatment of acute stroke [@doi10.1136/jnnp.2004.054882], rheumatoid arthritis [@doi:10.3899/jrheum.090074], and several hereditary, systemic or local inflammatory diseases [@doi:10.1056/NEJMoa055137, @doi:10.1186/s12969-017-0147-3, @doi:10.3389/fphar.2018.01157].
```

Comment:
```suggestion
Its primary function is to reduce inflammation by mediating IL-1, providing a broad spectrum of applications that include the treatment of acute stroke [@doi10.1136/jnnp.2004.054882], rheumatoid arthritis [@doi:10.3899/jrheum.090074], and several hereditary, systemic or local inflammatory diseases [@doi:10.1056/NEJMoa055137, @doi:10.1186/s12969-017-0147-3, @doi:10.3389/fphar.2018.01157].
```
@RLordan's suggestion that got lost while I was trying to figure out how to merge in all the suggestions!